guides/FRONTEND.md

```diff
@@ -22,7 +22,7 @@ yarn
 To connect to production servers, you'll need to create a file `.env.local` under `packages/web` and write this inside:
```

Comment:
you changed `NEXT_PUBLIC_GRAPHQL_URL` to use the staging backend, while this line and `.env.sample` still refer to the production backend (api.metagame.wtf). all of them should refer to the same backend instance.

Reply:
good job. updated :)

Reply:
line 22 in `FRONTEND.md` still refers to "production servers" :P

Reply:
updated